Makefrag

```diff
@@ -19,8 +19,9 @@ JVM_MEMORY ?= 2G
 SBT ?= java -Xmx$(JVM_MEMORY) -Xss8M -XX:MaxPermSize=256M -jar $(base_dir)/sbt-launch.jar
 SHELL := /bin/bash
 
+ROCKET_CLASSES := $(base_dir)/target/scala-2.11/classes
```

Comment:
This was desperately needed... One optional suggestion: add the Chisel class path and Chisel jar to enable in-line compilation from Firrtl passes. Granted, that is technically outside the scope of this PR and could be added separately.

e.g.: 
```make
ROCKET_CLASSES := $(base_dir)/target/scala-2.11/classes:$(base_dir)/chisel3/target/scala-2.11/*
```

Reply:
This makes total sense. At the *very* least it seems like a good idea to s/:=/?=.